datasets/glossis/glossis.py

```diff
@@ -34,50 +36,68 @@ def list_blobs(bucket_name, folder_name):
     return blobs
 
 
-def upload_to_gee(filename, bucket, asset):
+def wait_gee_tasks(tasks):
+    for task in task:
+        wait_gee_task(task)
+
+
+def wait_gee_task(task):
+    gee_cmd = "earthengine --service_account_file {creds} --no-use_cloud_api task wait {task}".format(
```

Comment:
Consider returning an [awaitable](https://docs.python.org/3/library/asyncio-task.html#awaitables).

Reply:
Agree that it could be used, see here: https://docs.python.org/3/library/asyncio-subprocess.html for more details. But since --wait gives us the same functionality and rewriting to async processes has no benefits here, I'd like to leave it as is.